devices/Sim7080/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("IoT.Device.Sim7080")]
+[assembly: AssemblyCompany("nanoFramework Contributors")]
+[assembly: AssemblyProduct("IoT.Device.Sim7080")]
+[assembly: AssemblyCopyright("Copyright © nanoFramework Contributors 2023")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM components.  If you need to access a type in this assembly from 
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+/////////////////////////////////////////////////////////////////
```

Comment:
The bindings libraries don't use Interop libraries. 
You have to remove this section enterily.

Reply:
Removed!